README.md

```diff
@@ -119,6 +119,14 @@
 - [OrdinalSafe](https://ordinalsafe.xyz)
 - [XDEFI Wallet](https://www.xdefi.io/)
 
+### Reddit Pages 
```

Comment:
I believe the correct term is "Subreddit" instead of "Page", can you edit this line please

Reply:
true, gonna change it